templates/_base.html.twig

```diff
@@ -2,18 +2,24 @@
 <html lang="en-GB">
   <head>
     <title>{{ subject }}</title>
-    <style type="text/css">
-    p {
-      font-size: 1.2rem;
-    }
-    </style>
   </head>
 <body>
-  <div id="content">{% block content %}{% endblock %}</div>
-  <div id="footer">
-    {% block footer %}
-        The Big Give xx
-    {% endblock %}
+  <div style="margin: 0 auto; max-width:500px;">
+    <img src="{{ header }}" alt="TheBigGive" title="The Big Give" width="250" height="50"/>
```

Comment:
If we're fixing the width at 250px we should serve the image at (at most) 500px for high-DPI displays. Also not sure what the reason is for scaling this way instead of with responsive CSS?